### DISCUSSION TITLE
indiscriminately show the python version

### DISCUSSION THREAD

**Discussion by bwagner** [starship/starship#6243] [Q&A]
This should be really simple, but I failed to find a way to **indiscriminately show the python version**, so, even when I'm in an empty directory.
`detect_files` was the closest configuration I found , but apparently, it can't be set to a [wildcard](https://starship.rs/config/#python:~:text=detect_files%20%3D%20%5B%27foo%27%5D%20%23%20can%20specify%20filters%20but%20wildcards%20are%20not%20supported) `"*"`.
The python version is only shown when there's a python file in the current directory.

 `starship.toml` for now without `[python]`-section, as changes didn't help.
```
# Get editor completions based on the config schema
"$schema" = 'https://starship.rs/config-schema.json'

# Inserts a blank line between shell prompts
add_newline = true

# Replace the '❯' symbol in the prompt with '➜'
[character] # The name of the module we are configuring is 'character'
success_symbol = '[➜](bold green)' # The 'success_symbol' segment is being set to '➜' with the color 'bold green'

# Disable the package module, hiding it from the prompt completely
[package]
disabled = true
```
starship version:
```
starship 1.20.1
branch:
commit_hash:
build_time:2024-07-27 09:07:23 +00:00
build_env:rustc 1.79.0 (129f3b996 2024-06-10) (Homebrew),
```
macOS 14.6.1 (Sonoma)

See also
https://github.com/starship/starship/discussions/2364

**Comment by davidkna** [comment]
Once [starship v1.21.0](https://github.com/starship/starship/pull/6146) is released with #4486, something like this could work:
```toml
[python]
detect_env_vars = ["PATH"]
```
